package.json

```diff
@@ -27,7 +27,7 @@
     "npm-check": "5.9.2",
     "papaparse": "5.2.0",
     "prop-types": "15.7.2",
-    "qrcode.react": "1.0.0",
+    "qrcode.react": "^1.0.0",
```

Comment:
```suggestion
    "qrcode.react": "1.0.0",
```

Reply:
Honestly I don't know why there's a change in this file, I didn't modify it...